Minio/ApiEndpoints/ObjectOperations.cs

```diff
@@ -118,20 +123,73 @@ public IObservable<Upload> ListIncompleteUploads(ListIncompleteUploadsArgs args,
         /// <summary>
         /// Get list of multi-part uploads matching particular uploadIdMarker
         /// </summary>
-        /// <param name="args">ListIncompleteUploadsArgs Arguments Object which encapsulates bucket name, prefix, recursive</param>
+        /// <param name="args">GetMultipartUploadsListArgs Arguments Object which encapsulates bucket name, prefix, recursive</param>
         /// <param name="cancellationToken">Optional cancellation token to cancel the operation</param>
         /// <returns></returns>
         private async Task<Tuple<ListMultipartUploadsResult, List<Upload>>> GetMultipartUploadsListAsync(GetMultipartUploadsListArgs args,
                                                                                      CancellationToken cancellationToken)
         {
             args.Validate();
             IRestResponse response = null;
-            RestRequest request = await this.CreateRequest(args).ConfigureAwait(false);
-            response = await this.ExecuteAsync(this.NoErrorHandlers, request, cancellationToken).ConfigureAwait(false);
+            try
+            {
+                RestRequest request = await this.CreateRequest(args).ConfigureAwait(false);
+                response = await this.ExecuteAsync(this.NoErrorHandlers, request, cancellationToken).ConfigureAwait(false);
+            }
+            catch (Exception ex)
+            {
+                if (ex.GetType() == typeof(BucketNotFoundException))
```

Comment:
Why are we ignoring this error here?

Reply:
This is an error encountered when procuring the list of incomplete uploads. If the bucket has been removed, the uploads have been removed as well, along with it. I encountered it while running the tests.